client/src/cmdhfmf.c

```diff
@@ -4625,19 +4638,20 @@ static int CmdHF14AMfCGetSc(const char *Cmd) {
 static int CmdHF14AMfCSave(const char *Cmd) {
     CLIParserContext *ctx;
     CLIParserInit(&ctx, "hf mf csave",
-                  "Save magic gen1a card memory into three files (BIN/EML/JSON)"
-                  "or into emulator memory",
-                  "hf mf csave\n"
-                  "hf mf csave --4k"
+                  "Save magic gen1a card memory into three files (BIN/EML/JSON) and/or into emulator memory\n"
+                  "Only works with magic gen1a cards",
+                  "hf mf csave                 --> Save a MIFARE 1K card to file using the UID as the filename\n"
```

Comment:
here the help text should explain the  UID as the filename and not the individual sample descriptions.